data-model-layer/data-model.md

```diff
@@ -113,7 +114,8 @@ Values contained in lists can be accessed by their ordinal offset in the list.
 Map is a recursive kind.
 
 Values in maps are accessed by their "key".  Maps can also be iterated over,
-yielding key+value pairs.
+yielding key+value pairs, and such iteration shall not assume any order to
```

Comment:
oh, this is treading into contested waters here, I don't know if we have agreement yet about map ordering while in the data model, we still don't even have agreement about the _right_ way to view ordering at the codec layer.

This _might_ be safe to say, I'd certainly want to reserve this property for some scenarios, but I don't think we can assert that there isn't "any order" (in the Go sense of "no stable ordering for you!").